packages/@aws-cdk/aws-lambda/lib/code.ts

```diff
@@ -160,6 +162,54 @@ export class InlineCode extends Code {
   }
 }
 
+/**
+ * Bundle options
+ */
+export interface BundleOptions {
+  /**
+   * The Docker image where the command will run.
+   */
+  readonly image: DockerImage;
```

Comment:
How about making [`lambci/lambda:build-${runtime}`](https://hub.docker.com/r/lambci/lambda/) the default image. That would be really cool.

Reply:
Also thought about this. The `Code` binds to a `cdk.Construct`, how would you elegantly check if it's a `Function`?

Reply:
`instanceof`